.github/workflows/ci.yml

```diff
@@ -0,0 +1,276 @@
+---
+name: Continuous Integration tests
+
+on: [pull_request]
+
+jobs:
+  main_tests:
+    name: Main tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      # Duplicate changes to this matrix to 'poc_tests'
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: ['3.9']
+        # Possible values: ['2.7', '3.6', '3.7', '3.8', '3.9', '3.10.0-rc.2']
+        include:
+          - os: ubuntu-latest
+            python-version: '3.6'
+          - os: ubuntu-latest
+            python-version: '3.7'
+          - os: ubuntu-latest
+            python-version: '3.8'
+          ## Expected failure
+          - os: ubuntu-latest
+            python-version: '3.10.0-rc.2'
```

Comment:
Failing main tests: https://github.com/dralley/hpy/pull/1/checks?check_run_id=3551010532#step:6:42

Reply:
ok, then I propose to just skip 3.10 for now